UI/src/utils/backend-utils.js

```diff
@@ -197,22 +203,18 @@
       });
     };
 
-    this.getAsset = function (assetId) {
-      self.getAssetWithCallback(assetId, function (asset) {
+    this.getAsset = function(nationalId) {
+      self.getAssetByExternalId(nationalId, function(asset) {
         eventbus.trigger('asset:fetched', asset);
       });
     };
 
-    this.getAssetWithCallback = function(assetId, callback) {
-      $.get('api/assets/' + assetId, callback);
-    };
-
-    this.getAssetByExternalId = function (externalId, callback) {
-      $.get('api/assets/' + externalId + '?externalId=true', callback);
+    this.getAssetByExternalId = function(nationalId, callback) {
```

Comment:
Maybe it would be better to rename 'getAssetByExternalId' as 'getAssetByNationalId'  
(or 'getMassTransitStopByNationalId')